src/engine/GuidanceLine.cs

```diff
@@ -76,6 +97,7 @@ public override void _Ready()
         var material = new StandardMaterial3D();
         material.VertexColorUseAsAlbedo = true;
         MaterialOverride = material;
+        yOffset = random.NextFloat() - 2.0f;
```

Comment:
Is this visible? What I mean that would it be safer to multiply the offset by 0.5f here to make the range half as big to maybe avoid some visual artifacts.

I tested in game and I didn't really notice anything, so I'll just comment this for future reference.